.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,200 @@
+
+<br>
+
+<details>
+<summary>Join the Bee Initiative :deciduous_tree:</summary>
+<br>
+
+The [Bee Initiative](https://github.com/iota-community/bee) is a collaborative effort to improve the Bee developer experience by focussing on the following goals:
+
+- Quality assurance and review
+- Documentation
+- Code samples
+- Improvements to modules and libraries
+
+## How much time is involved
+
+You can invest as much or as little time as you want into the initiative.
+
+## What's in it for you
+
+In return for your time, not only do you get to be a part of the future of IOTA technology, you will also be given a badge on Discord to show others that you're a valuable member of the IOTA community.
+
+## How to join
+
+If you're interested in joining, chat to us in the #experience channel on [Discord](https://discord.iota.org/).
+
+</details>
+
+<br>
+
+<details>
+<summary>Pull requests :mega:</summary>
+<br>
+
+This section guides you through submitting a pull request (PR). Following these guidelines helps give your PR the best chance of being approved and merged.
+
+### Before submitting a pull request
+
+Before submitting a pull request, please follow these steps to have your contribution considered by the maintainers:
+
+- A pull request should have exactly one concern (for example one feature or one bug). If a PR addresses more than one concern, it should be split into two or more PRs.
+
+- A pull request can be merged only if it references an open issue
+
+    **Note:** You don't need to open an issue for minor changes such as typos, but you can if you want.
+
+- All public interfaces should have descriptive documentation, including an
+example that compiles and passes [documentation tests](https://doc.rust-lang.org/rustdoc/documentation-tests.html)
+
+- All instances of `unsafe` should have a comment that explains why its use was unavoidable
+
+- Try to keep code comments to a minimum. If code needs to be commented to
+explain its function it can probably be refactored and be made more simple.
+
+- All code should be well tested, using unit tests and integration tests
+
+     `rustfmt` is the canonical source of truth for formatting. We use the default formatting options as defined by `rustfmt`, except for `edition = "2018"`.
+
+    If you want to prevent rustfmt from formatting code you have organized by hand(say macros, attributes, or lookup tables), you can do so by using the attribute `#[rustfmt::skip]`. A more finegrained configuration is possible, see the [rustfmt tips](https://github.com/rust-lang/rustfmt#tips).
+
+- Code must compile and pass tests for `x86_64` [Rust Tier 1 platforms](https://forge.rust-lang.org/platform-support.html). Testing will be done against Linux, macOS, and Windows operating systems.
+
+    We use the default `rustc` lints, except for the ones mentioned below. To not interfer with development, these lints are merely warnings. In PRs these warnings will be ignored, using the `RUSTFLAGS="-D warnings"` setting.
+
```

Comment:
This is not quite true. On the contrary we have a quite strict CI. You can probably put a link to the GH workflows repository.